DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs

```diff
@@ -426,12 +399,13 @@ public NameValueCollection Settings
                 }
 
                 // cloudservices variables need to be set regardless
-                this.settings.Add("cloudServices_tokenUrl", "/API/CKEditorProvider/CloudServices/GetToken");
+                var tokenUrl = ServicesFramework.GetServiceFrameworkRoot() + "API/CKEditorProvider/CloudServices/GetToken";
+                this.settings.Add("cloudServices_tokenUrl", tokenUrl);

```

Comment:
Meh, my commits got mixed up, the end result is still fine but this is basically the fix

Reply:
And thanks to @bdukes for the hint on the simpler solution than I had in head 😄